### DISCUSSION TITLE
Can´t access a function from inside nested loop.

### DISCUSSION THREAD

**Discussion by zzseba78** [alpinejs/alpine#1611] [7. Bugs (archived)] [answered]
I´m listing treatments and sub treatment for each treatment, everything works fine, but when i call a function from inside the nested loop on a link,  throws error `Uncaught (in promise) ReferenceError: **functionname** is not defined`

Of course the function is defined in a data object and i can access it from outside the nested loop and in the loop too, but not on the nested loop element, can someone point me in the right direction of what i´m missing? thanks!

**Comment by ryangjchandler** [comment, marked as answer]
Please provide a code snippet, otherwise it's difficult to help.

**Reply by zzseba78** [reply]
Sorry, i create a codpen and realize that´s not the cause of my problem.

https://codepen.io/zzseba78/pen/xxqoyoq
